src/runtime_src/CMakeLists.txt

```diff
@@ -62,6 +62,7 @@ endif()
 
 # Attach to the user's linker flags
 set (CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-Bsymbolic")
+set (CMAKE_SHARED_LINKER_FLAGS "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-z,defs")
```

Comment:
Does -z defs include the semantics of -Bsymbolic? We used the latter so our internal calls to OCL APIs were resolved within our library and not redirected to ICD.

Reply:
I am not aware of  zdef affecting anything but to enforce that libraries we build are fully resolved.